app/addons/cors/components.js

```diff
@@ -334,6 +334,7 @@ var CORSController = React.createClass({
             <br />
             <button
               type="button"
+              id="bt-enable-disable-cors"
```

Comment:
Ok cool, could you change it to a class and make it follow the BEM style we have for other components.